Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.3.0'
 
```

Comment:
In other words, update the Gemfile to 2.3.1, and set Travis to only run on 2.3.1